### PR TITLE
upgrade Makefile's tool versions to the latest

### PR DESCRIPTION
**Pull request by gazarenkov**

### CODE REVIEW

**Comment by nickboldt** [comment]
Please also update the dockerfiles and .yaml files in the .rhdh folder to align with these changes.

**Comment by gazarenkov** [comment]
> Please also update the dockerfiles and .yaml files in the .rhdh folder to align with these changes.

There are no diff with upstream/main after make bundle and visually nothing to update sync with .rhdh as well.

**Comment by gazarenkov** [comment]
> Please also update the dockerfiles and .yaml files in the .rhdh folder to align with these changes.

Looks like there are nothing to merge with .rhdh

**Comment by nickboldt** [comment]
>  Looks like there are nothing to merge with .rhdh

See these updates: https://github.com/redhat-developer/rhdh-operator/pull/67/commits/aaedb5d35d86ab4e4f5e28d20702d8b86c175908

Optionally you might also like this: to better align this "upstream RHDH operator" with the one we build downstream. https://github.com/redhat-developer/rhdh-operator/pull/67/commits/57e228cd31905e8602c866b3d09f3f33415e3989

I could also copy over more changes between the versions but we can do that in another PR... 

image

image

image

image

etc.

File: bundle/metadata/annotations.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ annotations:
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: backstage-operator
   operators.operatorframework.io.bundle.channels.v1: alpha
-  operators.operatorframework.io.metrics.builder: operator-sdk-v1.36.0
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.33.0
```
Comment (nickboldt) [inline bundle/metadata/annotations.yaml]: it seems you're using an older version of the SDK (1.33) compared to the version used in the commmit that followed this one.

**Review by nickboldt — APPROVED** [review]

**Pull request merged**